dicom_numpy/combine_slices.py

```diff
@@ -238,4 +238,4 @@ def _slice_spacing(slice_datasets):
 
 def _sort_by_slice_spacing(slice_datasets):
```

Comment:
I realize this wasn't part of the review, but the name of this function `_sort_by_slice_spacing` and the variable `slice_spacing` seem incorrect. Perhaps they be `_sort_by_slice_position` and `slice_positions` (note the "s") respectively?